src/build/mod.rs

```diff
@@ -1,5 +1,5 @@
 use crate::settings::toml::{Target, TargetType};
-use crate::terminal::{message, styles};
+use crate::terminal::{message, message::Message, styles};
```

Comment:
nit:
```suggestion
use crate::terminal::message::{Message, StdOut};
use crate::terminal::styles;
```
bums me out that you need the trait in scope 🤦‍♀️

Reply:
agreed, its unfortunate